runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/retries/StandardRetryStrategyTest.kt

```diff
@@ -252,7 +252,7 @@ class StringRetryPolicy : RetryPolicy<String> {
         "fail" -> RetryDirective.TerminateAndFail
         "client-error" -> RetryDirective.RetryError(RetryErrorType.ClientSide)
         "server-error" -> RetryDirective.RetryError(RetryErrorType.ServerSide)
-        "timeout" -> RetryDirective.RetryError(RetryErrorType.Timeout)
+        "timeout" -> RetryDirective.RetryError(RetryErrorType.Transient)
```

Comment:
**Nit**: For consistency, we should probably update `"timeout"` in this file to `"transient"`.